test/integration/vaultstaticsecret_integration_test.go

```diff
@@ -546,6 +572,17 @@ func TestVaultStaticSecret(t *testing.T) {
 	}
 }
 
+// getEnvInt reads an integer value from an environment variable.
+// If the env variable is not set or if it's not a valid integer, it returns the default value.
+func getEnvInt(key string, defaultValue int) int {
+	if value, exists := os.LookupEnv(key); exists {
+		if intValue, err := strconv.Atoi(value); err == nil {
+			return intValue
+		}
+	}
+	return defaultValue
+}
+
```

Comment:
Since `getEnvInt()` is used by other `vautl*_test.go`'s, could we refactor `getEnvInt()` into `integration_test.go` or `testing_util.go`?